docs/reference.rst

```diff
@@ -145,6 +145,10 @@ For more information about using casync support of RAUC, refer to
   By default, the chunk store path is derived from the location of the RAUC
   bundle you install.
 
+``tmpdir``
```

Comment:
For consistency with the other options, I'd prefer tmppath:
```suggestion
``tmppath``
```

Reply:
Done.

However, I kept a reference named `tmpdir` to reflect the `casync` environment variable `TMPDIR`.

See function `static gboolean casync_extract(RaucImage *image, gchar *dest, const gchar *seed, const gchar *store, const gchar *tmpdir, GError **error).`

This is identical to `storepath` that become `--store`.